WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
No, we shouldn't solve this by other means. There is no rule that a head of religion must be a duchy. For Melchisedechianism, this makes sense and is more accurate to the title and its history.